### PR TITLE
Add Statsig detection

### PR DESCRIPTION
**Pull request by ysm-dev**
https://statsig.com

- https://statsig.com
- https://tempo.fit/
- https://www.zleague.gg/

### CODE REVIEW

File: src/technologies/s.json [hunk 1]
```diff
@@ -5629,6 +5629,24 @@
     "scriptSrc": "cdn\\.statically\\.io/",
     "website": "https://statically.io"
   },
+  "Statsig": {
+    "cats": [
+      10
```
Comment (nurbek91) [inline src/technologies/s.json:5634]: We have a category also
85 Feature management

File: src/technologies/s.json [hunk 2]
```diff
@@ -5629,6 +5629,24 @@
...
+    ],
+    "description": "Statsig is a modern product observability platform that empowers teams to 10x their experimentation velocity, shipping every feature in a data-driven way.",
```
Comment (nurbek91) [inline src/technologies/s.json:5636]: Statsig is a modern product experimentation platform that helps product teams continuously measure impact of every single feature they launch.

File: src/technologies/s.json (hunk 1 above)
Comment (ysm-dev) [reply src/technologies/s.json]: [b700220](https://github.com/wappalyzer/wappalyzer/pull/7393/commits/b700220450bbea09dd9812cfecc4ff95cca3a993)

File: src/technologies/s.json (hunk 2 above)
Comment (ysm-dev) [reply src/technologies/s.json]: [b700220](https://github.com/wappalyzer/wappalyzer/pull/7393/commits/b700220450bbea09dd9812cfecc4ff95cca3a993)

File: src/technologies/s.json [hunk 3]
```diff
@@ -5629,6 +5629,25 @@
     "scriptSrc": "cdn\\.statically\\.io/",
     "website": "https://statically.io"
   },
+  "Statsig": {
+    "cats": [
+      10,
+      85
+    ],
+    "description": "Statsig is a modern product experimentation platform that helps product teams continuously measure impact of every single feature they launch.",
+    "icon": "Statsig.svg",
```
Comment (nurbek91) [inline src/technologies/s.json:5638]: Statsig
32x32

Comment (ysm-dev) [reply src/technologies/s.json:5638]: [0afb18d](https://github.com/wappalyzer/wappalyzer/pull/7393/commits/0afb18d5d24d034a70afe0b2352dbeec84fcfb6c)

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**